tests/test_autotuner.py

```diff
@@ -98,12 +73,10 @@ def test_timesteps_per_at_run():
     infield.data[:] = np.arange(reduce(mul, shape), dtype=np.int32).reshape(shape)
     outfield = Function(name='outfield', grid=grid)
     stencil = Eq(outfield.indexify(), outfield.indexify() + infield.indexify()*3.0)
-    op = Operator(stencil, dle=('blocking', {'blockalways': True}))
+    op = Operator(stencil, dle=('blocking', {'openmp': False, 'blockalways': True}))
     op(infield=infield, outfield=outfield, autotune=True)
-    out = [i for i in buffer.getvalue().split('\n') if 'took' in i]
-    assert len(out) == 4
-    assert all('in 1 timesteps' in i for i in out)
-    buffer.truncate(0)
+    assert op._state['autotuning'][-1]['runs'] == 4
+    assert op._state['autotuning'][-1]['tpr'] == 1
 
     # TimeFunction with increasing time order; increasing the time order
     # shouldn't affect how many iterations the autotuner is gonna run
```

Comment:
*going to* instead of gonna should be more formal ;-)

Reply:
code comments don't need to be formal.
API docstrings.... yeah